compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -141,10 +141,10 @@ gather_builtincode_map(const ::tflchef::ModelRecipe &model_recipe)
 
   for (const auto &operation : model_recipe.operation())
   {
-    auto op_chef = op_chef_registry().lookup(operation.type()).create(&operation);
-    if (op_chef->code() == tflite::BuiltinOperator_CUSTOM)
+    if ((operation.has_extype() && operation.extype() == "Custom") || operation.type() == "Custom")
```

Comment:
optional) as `type` is key and `extype` will be removed.
```suggestion
    if (operation.type() == "Custom" || (operation.has_extype() && operation.extype() == "Custom"))
```